SendNow/SettleGroup/ViewController/SettleView/GroupManagementViewController.swift

```diff
@@ -0,0 +1,239 @@
+//
+//  GroupManagementViewController.swift
+//  SendNow
+//
+//  Created by 한소희 on 11/29/25.
+//
+
+import Foundation
+import UIKit
+import RxSwift
+import Toast
+
+final class GroupManagementViewController: BaseUIViewController {
+    private let groupManagementView = GroupManagementView()
+    private let settleGroupViewModel: SettleGroupViewModel
+    private let disposeBag = DisposeBag()
+    
+    init(settleGroupViewModel: SettleGroupViewModel = SettleGroupViewModel(
+        userID: UserDefaults.standard.integer(forKey: MemberInfoField.userID.rawValue)),
+         groupID: Int) {
+        self.settleGroupViewModel = settleGroupViewModel
+        super.init(nibName: nil, bundle: nil)
+        self.settleGroupViewModel.setGroupID(groupID)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        settleGroupViewModel.loadGroupMemberInformation()
+        configureGroupManagementViewController()
+        addSubviews()
+        setLayoutConstraintsGroupManagementViewController()
```

Comment:
`GroupManagementViewController`는 생략해도 될 것 같아용
```suggestion
        configureGroup()
        addSubviews()
        setLayoutConstraints()
```